models/news-models.js

```diff
@@ -45,8 +46,43 @@ const fetchArticlesById = (article_id) => {
     })
 }
 
+const checkCommentExists = (article_id) => {
```

Comment:
Are you checking the comments exits, or that the article exists? The name of the function and its functionality don't match up here.